modules/admin_manual/pages/configuration/integration/ms-teams.adoc

```diff
@@ -6,10 +6,11 @@
 :teams-app-setup-policies-url: https://docs.microsoft.com/en-us/microsoftteams/teams-app-setup-policies
 :manage-apps-url: https://docs.microsoft.com/en-us/MicrosoftTeams/manage-apps
 :what-are-tabs-url: https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/what-are-tabs
+:msteams-generator-url: https://msteamsgen.owncloud.com
 
 == Introduction
 
-If you're using Microsoft Teams in your organization or for private purposes, you will likely want to access your ownCloud installation from your Microsoft Teams account. For this purpose, we created the {appsource-url}[Microsoft AppSource] app *ownCloud Generator for Admins* (expected to be available from end of August 2021) with which you can generate a customized Microsoft Teams app for your users accessing your ownCloud services. Each ownCloud domain accessed requires a separate generated Microsoft Teams app for your users. The generated app will be available in your organization's app catalog.
+If you're using Microsoft Teams in your organization or for private purposes, you will likely want to access your ownCloud installation from your Microsoft Teams account. For this purpose, we created the {msteams-generator-url}[ownCloud Generator for Admins] with which you can generate a customized Microsoft Teams app for your users accessing your ownCloud services. Each ownCloud domain accessed requires a separate generated Microsoft Teams app for your users. The generated app will be available in your organization's app catalog.
```

Comment:
> The generated app will be available in your organization's app catalog.

This reads as there is an automatism behind and conflicts with the statement:
"Go to the app section of Microsoft Teams and upload the generated zip file..."

Needs an improved text

Reply:
good point, changed it: https://github.com/owncloud/docs/pull/4045/commits/d56a6ce52f6acb8e235c5c5eddd53ba7b9492e17